Makefile

```diff
@@ -581,7 +592,9 @@ move: move_daemon move_freq move_appdaemon
 	@chmod 755 $(LIKWIDFILTERPATH)
 	@cp -f $(abspath $(PREFIX)/share/likwid/filter)/* $(LIKWIDFILTERPATH)
 	@chmod 755 $(LIKWIDFILTERPATH)/*
-	@install -m 644 $(LIBPREFIX)/likwid-config.cmake $(INSTALLED_LIBPREFIX)
+	@mkdir -p $(INSTALLED_LIBPREFIX)/cmake/likwid
+	@chmod 755 $(INSTALLED_LIBPREFIX)/cmake/likwid
+	@install -m 644 $(LIBPREFIX)/likwid-config.cmake $(INSTALLED_LIBPREFIX)/share/likwid
```

Comment:
shouldn't install be at `$(INSTALLED_LIBPREFIX)/cmake/likwid` instead of `$(INSTALLED_LIBPREFIX)/share/likwid`

Reply:
CMake has several paths it searches (that list I posted above) so either works. I chose the share path bc the directory is already made from other install commands

Reply:
The problem that I see is that in the `install` target the file is installed to `$(PREFIX)/share/likwid/likwid-config.cmake` but in `move` you use `$(LIBPREFIX)/likwid-config.cmake=$(PREFIX)/lib/likwid-config.cmake` as source for the file. The `move` target should move an already `install`ed LIKWID into another location. So, the cmake file probably needs some sed magic again to contain the proper paths in the end.

Reply:
Well I corrected not using `$(INSTALLED_LIBPREFIX)` and `$(INSTALLED_BINPREFIX)` in addition to adding the sed magic (which appears to not be necessary now).